crates/transaction-pool/src/error.rs

```diff
@@ -29,6 +30,15 @@ pub enum PoolError {
     /// respect the max_init_code_size.
     #[error("[{0:?}] Transaction's size {1} exceeds max_init_code_size {2}.")]
     TxExceedsMaxInitCodeSize(TxHash, usize, usize),
+    /// Thrown if a replacement transaction's gas price is below the already imported transaction
+    #[error("[{0:?}]: insufficient gas price to replace existing transaction.")]
+    AccountNotFound(TxHash),
+}
+
+impl From<Error> for PoolError {
+    fn from(_: Error) -> Self {
+        PoolError::AccountNotFound(TxHash::default())
+    }
 }
```

Comment:
why do we need this?
this does not seem right,
could remove I think?

Reply:
Yes this is not correct